experiments/findUncoveredAPI.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
Nice. Can you change the bash scripts to use `#!/usr/bin/env bash` as well?

Reply:
Sure. Should that be in this PR or a separate one?

Reply:
Let's do it in a separate PR